h2o-logger/src/main/java/water/logging/LoggerFactory.java

```diff
@@ -15,7 +20,21 @@
      * @return Slf4JLogger if SLF4J is on the classpath, ConsoleLogger if not
      */
     public static Logger getLogger(Class<?> clazz) {
```

Comment:
this method is not covered by tests, thats why having the static LoggerFactory INSTANCE is the best solution